OpenLDAP/openldap/_meta/smart-descriptions.json

```
@@ -0,0 +1,41 @@
+[
+  {
+    "value": "Establish a connection from {source.ip} to {destination.ip}",
+    "conditions": [
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.ip"
+      }
+    ]
+  },
+  {
+    "value": "Establish a connection from {source.ip}:{source.port} to {destination.ip}:{destination.port}",
+    "conditions": [
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "source.port"
+      },
+      {
+        "field": "destination.ip"
+      },
+      {
+        "field": "destination.port"
+      }
+    ]
+  },
```

I suggest adding additional smart-descriptions
```suggestion
  },
  {
    "value": "{user.name} was authenticated",
    "conditions": [
      {
        "field": "event.action",
        "value": "bind"
      },
      {
        "field": "user.name"
      }
    ]
  },
  {
    "value": "{user.name} was updated",
    "conditions": [
      {
        "field": "event.action",
        "value": "mod"
      },
      {
        "field": "user.name"
      }
    ]
  },
  {
    "value": "{openldap.attribute} was updated",
    "conditions": [
      {
        "field": "event.action",
        "value": "mod"
      },
      {
        "field": "openldap.attribute"
      }
    ]
  },
  {
    "value": "A search for domain {user.domain} was requested",
    "conditions": [
      {
        "field": "event.action",
        "value": "srch"
      },
      {
        "field": "user.domain"
      }
    ]
  },
  {
    "value": "A search for attribute {openldap.attribute} was requested",
    "conditions": [
      {
        "field": "event.action",
        "value": "srch"
      },
      {
        "field": "openldap.attribute"
      }
    ]
  },
```